storage/innobase/row/row0quiesce.cc

```diff
@@ -56,30 +56,24 @@ row_quiesce_write_index_fields(
 		mach_write_to_4(ptr, field->prefix_len);
 		ptr += sizeof(ib_uint32_t);
 
-		mach_write_to_4(ptr, field->fixed_len);
-
-		DBUG_EXECUTE_IF("ib_export_io_write_failure_9",
-				close(fileno(file)););
-
-		if (fwrite(row, 1, sizeof(row), file) != sizeof(row)) {
-
-			ib_senderrf(
-				thd, IB_LOG_LEVEL_WARN, ER_IO_WRITE_ERROR,
-				(ulong) errno, strerror(errno),
-				"while writing index fields.");
-
-			return(DB_IO_ERROR);
-		}
```

Comment:
The commit message had better mention this simplification. The surviving error message later in this function had better say "index fields" rather than "index column".